x/cdp/keeper/savings_test.go

```diff
@@ -76,7 +91,21 @@ func (suite *SavingsTestSuite) TestGetSetPreviousDistributionTime() {
 	pdt, f := suite.keeper.GetPreviousSavingsDistribution(suite.ctx)
 	suite.True(f)
 	suite.Equal(now, pdt)
+}
+
+func (suite *SavingsTestSuite) TestGetSetSavingsRateDistributed() {
+	// Savings rate dist set to 0 when the default genesis is used
+	preSavingsRateDistAmount := suite.keeper.GetSavingsRateDistributed(suite.ctx)
+	suite.True(preSavingsRateDistAmount.Equal(sdk.ZeroInt()))
```

Comment:
nit use the default variable instead

Reply:
https://github.com/Kava-Labs/kava/pull/644/commits/c682f2d721de29f4d36fb5c378f1a0fa99393970